src/components/Identity/addCustomerIdsToPayload.js

```diff
@@ -0,0 +1,24 @@
+import { assign } from "../../utils";
```

Comment:
Consider adding a method to `processCustomerIds` called `addToPayload(cookieJar, payload)`